quacc/custodian/qchem.py

```diff
@@ -0,0 +1,74 @@
+"""
+Custodian handlers for QChem
+"""
+from __future__ import annotations
+import sys
+
+from custodian import Custodian
+from monty.dev import requires
+
+from quacc import SETTINGS
+
+try:
+    import openbabel as ob
+except ImportError:
+    ob = None
+
+
+@requires(
+    ob,
+    "Openbabel must be installed. Try conda install -c conda-forge openbabel",
+)
+def run_custodian(
+    qchem_cores: int,
```

Comment:
I think it might be better to have this be `qchem_cores: int = multiprocessing.cpu_count()` (with `import multiprocessing` at the top that way there is a reasonable default always. But we just have to double-check it's getting the CPU count from the remote worker and not the local machine. I think it should be.

Reply:
Fixed. Will need to check and make sure it's working in practice once I start testing on LRC.